app/views/course/assessment/submission/submissions/_statistics.html.slim

```diff
@@ -14,7 +14,10 @@ div.panel.panel-default
           td = @submission.grade
         tr
           th = @submission.experience_points_record.class.human_attribute_name(:points_awarded)
-          td = @submission.points_awarded
+          - if can?(:grade, @submission)
+            = f.input :points_awarded, wrapper_tag: :td, label: false
```

Comment:
I know simple_form _will_ generate a `td` tag. I'm saying that Bootstrap expects input fields to be within a `div`, not a `td`. Replacing the wrapper removes the div.